### PR TITLE
[Doc]Update Java importer doc

### PR DESCRIPTION
**Pull request by jude-zhu**
update java importer doc

### CODE REVIEW

File: src/tools/importer/README.md [hunk 1]
```diff
@@ -1,33 +1,149 @@
...
+|--batch          | -b            | batch insert size.|2|
+|--name           | -n            | specify the space name.| myspace_test2 |
+|--schema         | -m            | specify the tag or edgetype name.| student |
+|--column         | -c            | properties of tag or edge to be inserted, separated by a comma | name,age |
+|--ranking        | -k            | the edge have ranking data. Default is false| true/false|
+|--file           | -f            | data file| ./tmp/data.txt |
+|--errorPath    | -d            | error log file | ./tmp/error.log |
+|--help           | -h            | list help||
+|--timeout        | -o            | specify connection timeout, in millisecond| 3000 |
+|--pswd           | -p            | graphd service password||
```
Comment (Amber1990Zhang) [inline src/tools/importer/README.md]: I think there is an unwanted `|` here although it won't do any harm to the table's looking.

File: src/tools/importer/README.md [hunk 2]
```diff
@@ -1,33 +1,149 @@
...
+|--connectionRetry       | -r            | connection retry times||
```
Comment (Amber1990Zhang) [inline src/tools/importer/README.md]: Ditto, please check.

File: src/tools/importer/README.md [hunk 3]
```diff
@@ -1,33 +1,149 @@
...
+|--executionRetry       | -e           | thrift execution retry times||
+|--stat           | -s            | print statistics info| true/false |
+|--type           | -t            | indicate to insert vertex properties or edge properties| vertext/edge|
+|--user           | -u            | graphd service username||
+
+> For edge type, if 'ranking' column is specified, '--ranking' options should be set to `true`.
```
Comment (Amber1990Zhang) [inline src/tools/importer/README.md]: option or options?

File: src/tools/importer/README.md [hunk 4]
```diff
@@ -1,33 +1,149 @@
...
+
+### Example
+
+The examples are based on the [Insert Data Section of get-started.md](../../../docs/get-started.md#insert-data).
```
Comment (Amber1990Zhang) [inline src/tools/importer/README.md]: I think `the` is unnecessary here.

File: src/tools/importer/README.md [hunk 5]
```diff
@@ -1,33 +1,149 @@
...
+
+
+E1. Insert vertices 200, 201, 202.
+
+datafile.txt
```
Comment (Amber1990Zhang) [inline src/tools/importer/README.md]: `datafile.txt` or `data.txt`?

File: src/tools/importer/README.md [hunk 6]
```diff
@@ -1,33 +1,149 @@
 
-#### Importer
+## Importer
 
-`Importer` is a tool use to load `CSV` file into `Nebula`.
+`Importer` is a tool used to load a small amount of data from a CSV file into Nebula Graph. It is single-threaded. If you have a large set of data, generating SST files and ingesting these files may be a better way. For details, please refer to [Spark SST File Generator](../spark-sstfile-generator/README.md) and HDFS Importer <!--to be done-->
```
Comment (Amber1990Zhang) [inline src/tools/importer/README.md]: How about condensing the first two sentences into one?
`Importer` is a single-threaded tool used to load small data set from a CSV file into Nebula Graph.
As for large data set, generating SST files and then ingesting them may be a better way.

File: src/tools/importer/README.md [hunk 7]
```diff
@@ -1,33 +1,149 @@
...
-user           | u            | thrift service username.
+#### Install JARs
+
+```
+ > mvn install:install-file -Dfile=$YOUR_GRAPH_CLIENT_FILEPATH -DgroupId=nebula-graph -DartifactId=graph-client -Dversion=$GRAPH_CLIENT_VERSION -Dpackaging=jar
+
+ > mvn install:install-file -Dfile=$YOUR_FBTHRIFT_FILEPATH -DgroupId=com.facebook -DartifactId=thrift -Dversion=$FBTHRIFT_VERSION -Dpackaging=jar
+```
+
+Replace "$YOUR_GRAPH_CLIENT_FILEPATH" and "$YOUR_FBTHRIFT_FILEPATH" with your own values. Also "$GRAPH_CLIENT_VERSION" and "$FBTHRIFT_VERSION".
```
[the remark this answers is not in the document]
Comment (darionyaphet) [reply src/tools/importer/README.md]: ${YOUR_GRAPH_CLIENT_FILEPATH}  maybe better ?

File: src/tools/importer/README.md [hunk 8]
```diff
@@ -1,33 +1,149 @@
...
+
+### Get Importer
+
+Get the `importer` jar package from oss https://nebula-graph.oss-cn-hangzhou.aliyuncs.com/jar-packages/importer-1.0.0-beta.jar.
+
+Or build it by yourself.
+
+```
+> cd $YOUR_NEBULA_DIR
```
Comment (darionyaphet) [inline src/tools/importer/README.md]: ditto

File: src/tools/importer/README.md (hunk 5 above)
Comment (darionyaphet) [reply src/tools/importer/README.md]: +1

File: src/tools/importer/README.md [hunk 9]
```diff
@@ -1,33 +1,149 @@
...
+200,101,0,5
+200,102,0,5
+201,102,0,3
+202,102,0,3
+```
+
+The importer command:
+
+```
+> java -jar importer-1.0.0-beta.jar --address 127.0.0.1:3699 --name myspace_test2 --schema select -u user -p password -t edge --file data.txt --column grade --batch 2 -k true
```
Comment (darionyaphet) [inline src/tools/importer/README.md]: importer-${VERSION}.jar

File: src/tools/importer/README.md [hunk 10]
```diff
@@ -1,33 +1,149 @@
...
+```
+
+> The "ranking" column is optional. It works with "--ranking" optional, which will be described below.
+
+### Import Data
+
+To import data, run:
+
+```
+> java -jar importer-1.0.0-beta.jar <OPTIONS>
```
Comment (darionyaphet) [inline src/tools/importer/README.md]: importer-${VERSION}.jar

File: src/tools/importer/README.md [hunk 11]
```diff
@@ -1,33 +1,149 @@
...
+```
+200,"Monica",16,"female"
+201,"Mike",18,"male"
+202,"Jane",17,"female"
+```
+
+The importer command:
+
+```
+> java -jar importer-1.0.0-beta.jar --address 127.0.0.1:3699 --name myspace_test2 --schema student -u user -p password -t vertex --file data.txt --column name,age,gender --batch 2
```
Comment (darionyaphet) [inline src/tools/importer/README.md]: importer-${VERSION}.jar

File: src/tools/importer/README.md [hunk 12]
```diff
@@ -1,33 +1,149 @@
...
+200,101,5
+200,102,5
+201,102,3
+202,102,3
+```
+
+The importer command:
+
+```
+> java -jar importer-1.0.0-beta.jar --address 127.0.0.1:3699 --name myspace_test2 --schema select -u user -p password -t edge --file data.txt --column grade --batch 2
```
Comment (darionyaphet) [inline src/tools/importer/README.md]: importer-${VERSION}.jar

File: src/tools/importer/README.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (Amber1990Zhang) [reply src/tools/importer/README.md]: > ${YOUR_GRAPH_CLIENT_FILEPATH} maybe better ?

Yes, goog idea.

File: src/tools/importer/README.md [hunk 13]
```diff
@@ -1,33 +1,151 @@
...
+ > mvn install:install-file -Dfile=${YOUR_GRAPH_CLIENT_FILEPATH} -DgroupId=nebula-graph -DartifactId=graph-client -Dversion=${GRAPH_CLIENT_VERSION} -Dpackaging=jar
+
+ > mvn install:install-file -Dfile=${YOUR_FBTHRIFT_FILEPATH} -DgroupId=com.facebook -DartifactId=thrift -Dversion=${FBTHRIFT_VERSION} -Dpackaging=jar
+```
+
+Replace "${YOUR_GRAPH_CLIENT_FILEPATH}" and "${YOUR_FBTHRIFT_FILEPATH}" with your own values. Also "${GRAPH_CLIENT_VERSION}" and "${FBTHRIFT_VERSION}".
+
+### Get Importer
+
+Get the `Importer` jar package from oss https://nebula-graph.oss-cn-hangzhou.aliyuncs.com/jar-packages/importer-1.0.0-beta.jar.
```
Comment (whitewum) [inline src/tools/importer/README.md]: cloud storage OSS:

    wget -o importer-1.0.0-beta.jar https://nebula-graph.oss-cn-hangzhou.aliyuncs.com/jar-packages/importer-1.0.0-beta.jar

File: src/tools/importer/README.md [hunk 14]
```diff
@@ -1,33 +1,151 @@
...
-You can use `./run.sh ${CONFIG_PARAMETER} ${JAVA_FBTHRIFT_JAR}` to startup it.
+`Importer` is a single-threaded tool used to load small data set from a CSV file into Nebula Graph.
+As for large data set, generating SST files and then ingesting them may be a better way. For details, please refer to [Spark SST File Generator](../spark-sstfile-generator/README.md) and HDFS Importer <!--to be done-->
 
-When loading vertex csv file, the first field is vid and then following the value list.
+### Download JARs
 
-The edge csv file's structure look like: 
+<!-- to be replaced after Nexus MVN repo being set -->
+Our maven repository is under construction. At this moment, get fbthrift and graph-client packages from oss https://nebula-graph.oss-cn-hangzhou.aliyuncs.com/jar-packages/thrift-1.0-SNAPSHOT.jar and https://nebula-graph.oss-cn-hangzhou.aliyuncs.com/jar-packages/graph-client-1.0.0-beta.jar.
```
Comment (whitewum) [inline src/tools/importer/README.md]: cloud storage OSS:
   
    wget ...

and 

    wget ...

**Pull request closed without merging**